pkg/cmd/cli/snapshotlocation/create.go

```diff
@@ -54,22 +55,27 @@ func NewCreateCommand(f client.Factory, use string) *cobra.Command {
 }
 
 type CreateOptions struct {
-	Name     string
-	Provider string
-	Config   flag.Map
-	Labels   flag.Map
+	Name       string
+	Provider   string
+	Config     flag.Map
+	Labels     flag.Map
+	Credential flag.Map
+	secretName string
```

Comment:
Looks like VSL's create and set command's Options are different.
secretName and secretKey are used as internal values to keep Credential's secret name and key.
Could these two parameters also removed in create command?

Reply:
@blackpiglet I've gone through the BSL credentials commits after the initial implementation (some refactoring and validation changes) -- making similar changes here removes these Options fields, as they're not really necessary.